tests/unit/lastuser_core/test_session_UserSession.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
+
+from lastuserapp import db, init_for
+import lastuser_core.models as models
+from .test_db import TestDatabaseFixture
+from datetime import datetime as dt
+from coaster.utils import buid
+from datetime import datetime, timedelta
+from flask.testing import FlaskClient
+from flask import url_for
+from bs4 import BeautifulSoup
+
+
+class TestUser(TestDatabaseFixture):
+
+    def test_UserSession_init(self):
+        """Test to verify the creation of UserSession instance"""
+        result = models.UserSession()
+        self.assertIsInstance(result, models.UserSession)
+
+    def test_usersession_ua(self):
+        """Test to verify user_agent property of UserSession instance"""
+        ua=u'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36'
+        another_user_session = models.UserSession(user_agent=ua)
+        self.assertIsInstance(another_user_session.ua, dict)
+
+    def test_usersession_has_sudo(self):
+        """Test to set sudo and test if UserSession instance has_sudo """
+        another_user_session = models.UserSession()
+        another_user_session.set_sudo()
+        self.assertTrue(another_user_session.has_sudo)
+
+    def test_usersession_revoke(self):
+        """Test to revoke on UserSession instance"""
+        crusoe = self.fixtures.crusoe
+        yet_another_usersession = models.UserSession(user=crusoe, ipaddr='192.168.1.1', user_agent=u'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36', accessed_at=datetime.utcnow())
+        yet_another_usersession.revoke()
+        result = models.UserSession.query.filter_by(user=crusoe).one_or_none()
+        assert(result.revoked_at) is not None
+
+    def test_UserSession_get(self):
+        """Test for verifying UserSession's get method"""
+        oakley = self.fixtures.oakley
+        oakley_buid = buid()
+        oakley_session = models.UserSession(user=oakley, ipaddr='192.168.1.2', buid=oakley_buid, user_agent=u'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36', accessed_at=datetime.utcnow())
+        result = oakley_session.get(buid=oakley_buid)
```

Comment:
`get` is a class method, so it typically will be used as `UserSession.get(<buid>)`.